cryptoconditions/condition.py

```diff
@@ -74,63 +94,98 @@ def from_uri(serialized_condition):
         Returns:
             Condition: Resulting object
         """
+        # TODO consider removing.
```

Comment:
Agree. Don't see the reason for it. The check to see if its a string should be enough